static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -122,13 +144,33 @@ const handleCancelContributionSubmit = (): void => {
                 reason: cancellationReason,
             },
         }
-    ).catch(err => {
-        hideLoader();
-        displayErrorMessage();
-        reportError(err, {
-            feature: 'mma-monthlycontribution',
+    )
+        .then(resp => {
+            hideContributionDetails();
+            hideCardDetails();
+            hideCancelContributionForm();
+            displaySupportUpSell();
+
+            if (resp.status === 200) {
+                displayCancelContributionSuccessMessage();
+            } else {
+                displayCancelContributionErrorMessage();
```

Comment:
could this else throw a second exception to be picked up on the catch below?

Reply:
Good call, thanks!